handlers/_swagger.js

```diff
@@ -319,31 +319,32 @@ function registerRoute(app, auth, additionalMiddleware, method, path, data, allo
             setDefaultQueryParams(req, data, logger);
             setDefaultHeaders(req, data, logger);
 
-            //Wrap the set function, which is responsible for setting headers
-            if (allowedTypes) {
-                //Validate that the content-type is correct per the swagger definition
-                wrapCall(res, 'set', function (name, value) {
-                    if (name === 'Content-Type') {
-                        var type = value.split(';')[0]; //parse off the optional encoding
-                        if (!_.contains(allowedTypes, type)) {
-                            logger.warn('Invalid content type specified: %s. Expecting one of %s', type, allowedTypes);
-                        }
+            // Wrap the set function, which is responsible for setting headers
+            var type = '';
```

Comment:
for clarity, I think it would be good to call the variable something like `responseContentType`